db/migrate/20171018132438_add_image_to_vendors.rb

```diff
@@ -0,0 +1,5 @@
+class AddImageToVendors < ActiveRecord::Migration[5.1]
+  def change
+    add_column :vendors, :image, :string
```

Comment:
Yes! Needed this.

Reply:
Nice, now we can revive Uncle Frank.

Reply:
I think we are obligated to revive Uncle Frank.